jwst/datamodels/schemas/slitmeta.schema.yaml

```diff
@@ -0,0 +1,95 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/slitmeta.schema"
+type: object
+  properties:
```

Comment:
I think you need to remove the 2 whitespaces at the beginning of this line, in order to get the syntax correct (and fix the unit test failures).

Reply:
I have been struggling trying to get this line fixed in flake8. I added allOf at the top is that needed ?
So properties should not have 2 white spaces ? All the other schemas seem to have this. 
But flake8 does not like  line 6

Reply:
Ok in the wcsinfo.schema there is no 'allOf' and there is not indenting of  of properties. I will try that format.